Update changelog for September 19 2024 (#3455)
timescale/docs: about/changelog.md

All the latest features and updates to Timescale products.

## Timescale Console, and SQL editor improvements
<Label type="date">September 19, 2024</Label>

### 🤝 New modes in Timescale Console: Ops and Data mode

We've been listening to your feedback and noticed that Timescale Console users have diverse needs. Some of you are focused on operational tasks like adding replicas or changing parameters, while others are diving deep into data analysis to gather insights.

**To better serve you, we've introduced new modes to the Timescale Console UI—tailoring the experience based on what you're trying to accomplish.**

Ops mode is where you can manage your services, add replicas, configure compression, change parameters, etc. 

Data mode is the full PopSQL experience: write queries with autocomplete, visualize data with charts and dashboards, schedule queries and dashboards to create alerts or recurring reports, share queries and dashboards, and more.

Try it today and let us know what you think! 

> ![Timescale Console Ops and Data mode]( !!! URL !!! )

Author: ```suggestion
![Timescale Console Ops and Data mode](https://s3.amazonaws.com/assets.timescale.com/docs/images/ops-data-mode.gif)
```

All the latest features and updates to Timescale products.

## 🤝 New modes in Timescale Console: Ops and Data mode, and Console based Parquet File Import

<Label type="date">September 19, 2024</Label>

We've been listening to your feedback and noticed that Timescale Console users have diverse needs. Some of you are focused on operational tasks like adding replicas or changing parameters, while others are diving deep into data analysis to gather insights.

**To better serve you, we've introduced new modes to the Timescale Console UI—tailoring the experience based on what you're trying to accomplish.**

Ops mode is where you can manage your services, add replicas, configure compression, change parameters, and so on.

Data mode is the full PopSQL experience: write queries with autocomplete, visualize data with charts and dashboards, schedule queries and dashboards to create alerts or recurring reports, share queries and dashboards, and more.

Try it today and let us know what you think! 

![Timescale Console Ops and Data mode](https://assets.timescale.com/docs/images/ops-data-mode.gif)

## Console based Parquet File Import

> * Now users can upload from Parquet  to Timescale by uploading the file from their local file system.  For files larger than 250 MB or if you want to do it yourself, follow the three-step process to upload Parquet files to Timescale.

Reviewer: ```suggestion
Now users can upload from Parquet to Timescale Cloud by uploading the file from their local file system. For files larger than 250 MB, or if you want to do it yourself, follow the three-step process to upload Parquet files to Timescale.
```

> https://github.com/user-attachments/assets/b1048bfe-e2a2-433a-a767-d7e00a20f715

Reviewer: ```suggestion
![Upload from Parquet to Timescale Cloud](https://assets.timescale.com/docs/images/upload-parquet.gif)
```

All the latest features and updates to Timescale products.

## 🤝 New modes in Timescale Console: Ops and Data mode, and Console based Parquet File Import

<Label type="date">September 19, 2024</Label>

We've been listening to your feedback and noticed that Timescale Console users have diverse needs. Some of you are focused on operational tasks like adding replicas or changing parameters, while others are diving deep into data analysis to gather insights.

**To better serve you, we've introduced new modes to the Timescale Console UI—tailoring the experience based on what you're trying to accomplish.**

Ops mode is where you can manage your services, add replicas, configure compression, change parameters, and so on.

Data mode is the full PopSQL experience: write queries with autocomplete, visualize data with charts and dashboards, schedule queries and dashboards to create alerts or recurring reports, share queries and dashboards, and more.

Try it today and let us know what you think! 

![Timescale Console Ops and Data mode](https://assets.timescale.com/docs/images/ops-data-mode.gif)

## Console based Parquet File Import

> * Now users can upload their Parquet files to Timescale by simply selecting the file from their local file system. 

Reviewer: ```suggestion
* Now users can upload from Parquet  to Timescale by uploading the file from their local file system.  For files larger than 250 MB or if you want to do it yourself, follow the three-step process to upload Parquet files to Timescale.
```

> * For files larger than 250 MB or if users prefer to do it themselves, they can follow a three-step process to upload Parquet files into Timescale.

Reviewer: Delete please

https://github.com/user-attachments/assets/b1048bfe-e2a2-433a-a767-d7e00a20f715

> ### Other improvements

Reviewer: ```suggestion
## SQL editor improvements
```

> * In the Ops mode SQL editor, you can now highlight a statement to just run that specific statement

Reviewer: ```suggestion
* In the Ops mode SQL editor, you can now highlight a statement to run a specific statement.
```
